### DISCUSSION TITLE
機器人的觸覺感受🖐

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#97] [General]
**新聞大綱：**
依照Facebook AI研究室的人員撰文內容表示，目前希望透過在機器人身上加上觸覺感知能力，使其能獲得更貼近人類的互動，能像人類透過皮膚觸覺感受物品質感、重量。
實際設計中，分別以可識別機器手臂指尖壓按深度的**DIGIT感測元件**，以及可模擬「觸覺」的ReSkin感測元件，讓機器人可以自主判斷機器手臂壓按力道程度。
其中，DIGIT感測元件可識別高解析度影像設計。當機器手臂指尖開始進行壓按動作時，系統就會配合預先訓練數據內容，透過影像識別方式判斷力道。
與卡內基梅隆大學合作的**ReSkin感測元件**中，則可模仿人類皮膚的設計，在彈性材質底下加入微磁粒子，搭配軟性電路板運作。透過磁力改變計算相關數據，讓機器手臂隨著施與力道增加而判斷物品受力程度。而2-3mm厚度設計的ReSkin約可對應5萬次使用壽命，約可在1mm高度形變過程中以400Hz識別率判斷施與力道，精準度更可達90%。
不過，兩種設計都必須透過深度學習方式訓練，意味會侷限在預先訓練使用資料範圍內，但預期未來將能透過不同學習框架，讓機器人可以學習知曉更多「觸覺」。
目前此類技術也將能應用在更多**智慧穿戴裝置**觸控互動，例如應用在衣物等軟性表面物品，藉此產生觸控操作反應，甚至應用在機器人身上也能模擬產生「痛覺」或「搔癢感」。
**新聞心得：**
能讓機器人以深度學習的方式使他擁有觸覺的感受，而且還可以自行判斷力道，感覺這項判別需要經過很長時間的訓練才能讓機器學會，兩個感測元件無論是功能還是組成都好高科技，也很期待這個研究室日後會讓機器人學習甚麼其他的觸感，或是未來五種知覺都發展出來的話應該會很有趣！
**新聞來源：**
https://mashdigi.com/teaching-robots-to-perceive-understand-and-interact-through-touch/

![247987901_566853651271831_951822478087944410_n](https://user-images.githubusercontent.com/90883526/140278927-faef5dbd-b415-409b-9383-1ec865316557.jpg)

**Comment by Jay-722** [comment]
機器人有觸覺也太奇怪了吧，他可以感受到毛茸茸的東西了

**Comment by pocper1** [comment]
所以機器人被揍，也能感到痛覺...? 感覺好奇怪XD

**Comment by minniewei** [comment]
哈哈哈，所以以後有人打機器人機器人還會覺得很痛然後逃跑！？真是太可愛了吧！

**Comment by wh20502** [comment]
其實連人類的觸覺都存在許多弔詭的事情，機器人如果也能夠具有觸覺，那麼就能夠多一項偵測環境的方式了。

**Comment by 110409538** [comment]
機器人有觸覺的話，感覺很奇妙。

**Comment by yasklop** [comment]
沒想到機器人連觸覺都有了，會不會哪天機器人能有嗅覺或味覺

**Comment by ghost** [comment]
機器人如果有觸覺 那他們如果直直走向牆壁撞到是不是會抱頭喊痛呢 XD

**Comment by Shiba010** [comment]
感覺機器人就算知道“痛”，應該本身也是不同不癢。
